applications/wg-easy/container/Containerfile

```diff
@@ -0,0 +1,68 @@
+# Base image for all shared Dockerfiles for taskfiles
```

Comment:
Why would this live in a subfolder "container" if it's the only thing in that sub-folder? 
Do you expect there to be scripts that are added in the future for entrypoints and such that will live here too?